cmd/olm/main.go

```diff
@@ -257,10 +258,30 @@ func main() {
 		go monitor.Run(op.Done())
 	}
 
+	// Emit CSV metric
+	if err = ensureCSVMetric(logger, crClient); err != nil {
+		logger.WithError(err).Fatalf("error emitting CSV metric")
```

Comment:
`nit`: similar to the other comment: "error emitting metrics for existing CSV" might be more readable.